util/admin/admin.go

```diff
@@ -490,12 +491,30 @@ func rowWithCols(sessCtx sessionctx.Context, txn kv.Retriever, t table.Table, h
 			}
 			continue
 		}
+		if col.IsGenerated() && col.GeneratedStored == false {
+			genFlag = true
+			break
```

Comment:
Why break here? If there are handle column after this column, the check would fail because it did set the handle as line 486 do.

Reply:
Add some comments here. Why break?

Reply:
I have remove break.